stan-native/src/main/java/org/ballerinalang/nats/streaming/consumer/Attach.java

```diff
@@ -78,6 +78,9 @@ private static boolean getAckMode(BObject service) {
                                                               Utils.getModule().getName() + VERSION_SEPARATOR +
                                                               Utils.getModule().getVersion() +
                                                               ":" + Constants.NATS_STREAMING_SUBSCRIPTION_ANNOTATION));
+        if (serviceConfig == null) {
```

Comment:
This throws a NullPointerException when attach() is called without a service config.